lua/extn/sky/unstable/io/arc.lua

```diff
@@ -46,24 +52,109 @@ function ArcDialGesture:new(props)
 end
 
 function ArcDialGesture:process(event, output, state)
-  if sky.is_type(event, ArcInput.ARC_ENC_EVENT) then
-    if event.n == self.which then
-      local range = self.max - self.min
-      local inc = range / self.steps
-      local change = inc * event.delta * self.scale
-      local next = util.clamp(self._value + change, self.min, self.max)
-      if next ~= self._value then
-        self._value = next
-        output({
-          type = self.ARC_DIAL_EVENT,
-          arc = event.arc,
-          n = event.n,
-          value = self._value,
-          normalized = self._value / range,
-        })
+  if sky.is_type(event, ArcInput.ARC_ENC_EVENT) and event.n == self.which then
```

Comment:
Should this check use ArcInput.is_enc()?

Reply:
good catch will update.